libsweep/protocol.h

```diff
@@ -147,8 +151,8 @@ inline void speed_to_ascii_bytes(int32_t speed, uint8_t bytes[2]) {
   // Speed values are still ASCII codes, numbers begin at code point 48
   const uint8_t ASCIINumberBlockOffset = 48;
 
-  uint8_t num1 = (speed / 10) + ASCIINumberBlockOffset;
-  uint8_t num2 = (speed % 10) + ASCIINumberBlockOffset;
+  uint8_t num1 = (uint8_t)(speed / 10) + ASCIINumberBlockOffset;
+  uint8_t num2 = (uint8_t)(speed % 10) + ASCIINumberBlockOffset;
```

Comment:
Because VS gives a warning here due to the narrowing conversion. But they should actually be `static_cast`s instead of c-style casts.